src/Svelte/composer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "symfony/ux-svelte",
+    "type": "symfony-bundle",
+    "description": "Integration of Svelte in Symfony",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Titouan Galopin",
+            "email": "galopintitouan@gmail.com"
+        },
+        {
+            "name": "Thomas Choquet",
+            "email": "thomas.choquet.pro@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Svelte\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\Svelte\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "symfony/webpack-encore-bundle": "^1.15"
+    },
+    "require-dev": {
+        "symfony/framework-bundle": "^4.4|^5.0|^6.0",
+        "symfony/phpunit-bridge": "^5.2|^6.0",
+        "symfony/twig-bundle": "^4.4|^5.0|^6.0",
+        "symfony/var-dumper": "^4.4|^5.0|^6.0"
```

Comment:
Use `^5.4|^6.2` for all of these. New package, so we can be more aggressive on the versions of Symfony we need to test with / support.